Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/FarePolicy.hs

```diff
@@ -74,19 +72,19 @@ validateUpdateFarePolicyRequest UpdateFarePolicyReq {..} =
       validateField "nightShiftEnd" nightShiftEnd . InMaybe $ InRange (TimeOfDay 0 30 0) (TimeOfDay 7 0 0)
     ]
 
-listFarePolicies :: (HasCacheConfig r, EsqDBFlow m r, HedisFlow m r) => SP.Person -> m ListFarePolicyRes
-listFarePolicies person = do
-  oneWayFarePolicies <- SFarePolicy.findAllByMerchantId person.merchantId
-  pure $
-    ListFarePolicyRes
-      { oneWayFarePolicies = map makeFarePolicyAPIEntity oneWayFarePolicies
-      }
+-- listFarePolicies :: (HasCacheConfig r, EsqDBFlow m r, HedisFlow m r) => SP.Person -> m ListFarePolicyRes
+-- listFarePolicies person = do
+--   oneWayFarePolicies <- SFarePolicy.findAllByMerchantId person.merchantId
+--   pure $
+--     ListFarePolicyRes
+--       { oneWayFarePolicies = map makeFarePolicyAPIEntity oneWayFarePolicies
+--       }
 
 updateFarePolicy :: (HasCacheConfig r, EsqDBFlow m r, CoreMetrics m, HedisFlow m r) => SP.Person -> Id DFarePolicy.FarePolicy -> UpdateFarePolicyReq -> m UpdateFarePolicyRes
 updateFarePolicy admin fpId req = do
   runRequestValidation validateUpdateFarePolicyRequest req
   farePolicy <- SFarePolicy.findById fpId >>= fromMaybeM NoFarePolicy
-  unless (admin.merchantId == farePolicy.merchantId) $ throwError AccessDenied
+  -- unless (admin.merchantId == farePolicy.merchantId) $ throwError AccessDenied
```

Comment:
Why?

Reply:
Yes, because we dont have merchantId stored in farepolicy anymore. Actually i will comment this APIs as it is not used anymore